### PR TITLE
NFL test

### PR DESCRIPTION
**Pull request by alissapajer**
This is the test from ch2110.

Verified that there is a single `LeftShift`! :tada:

### CODE REVIEW

File: it/src/main/resources/tests/shiftNFLTeams.test [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      "SELECT c.p6 AS A, c.p7 AS B, c.p8 AS C
+        FROM (SELECT pt1.g3 AS p6, pt1.g4 AS p7, pt1.g5 AS p8
+          FROM (SELECT (
+            SELECT * FROM (SELECT t.teams[_].team FROM `nfl-teams.data` AS t) AS v
+              WHERE type_of(v) = \"string\") AS g3, (
+            SELECT * FROM (SELECT t.teams[_].wins FROM `nfl-teams.data` AS t) AS v
+              WHERE type_of(v) = \"number\") AS g4, (
+            SELECT * FROM (SELECT t.teams[_].losses FROM `nfl-teams.data` AS t) AS v
+              WHERE type_of(v) = \"number\") AS g5
+            FROM `nfl-teams` AS gt2) AS pt1) AS c",
```
Comment (alissapajer) [inline it/src/main/resources/tests/shiftNFLTeams.test]: Ooops missing `.data`, pushing a change. Test passes without it though, so it's not a meaningful part of the sql2, though maybe sql requires it?

**Comment by alissapajer** [comment]
@djspiewak should I close this and reopen against master?

**Comment by djspiewak** [comment]
@alissapajer Depends on if you want to hold my backport merge until after this lands (at which point I would just recreate the backport merge to include this PR). Totally up to you.

**Comment by alissapajer** [comment]
I'd feel more comfortable having this in the backport branch, since it's proof that changes that landed in backport have the intended effects. 

@djspiewak want to merge this?

**Pull request merged**